crate/memories/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#[cfg(feature = "redis-mem")]
+mod redis_mem;
+#[cfg(feature = "redis-mem")]
+pub use redis_mem::{RedisMemory, RedisMemoryError};
+
+#[cfg(feature = "sqlite-mem")]
+mod sqlite_mem;
+#[cfg(feature = "sqlite-mem")]
+pub use sqlite_mem::{FINDEX_TABLE_NAME, SqliteMemory, SqliteMemoryError};
+
+#[cfg(feature = "postgres-mem")]
+mod postgresql_mem;
+#[cfg(feature = "postgres-mem")]
+pub use postgresql_mem::{PostgresMemory, PostgresMemoryError};
+
+pub mod reexport {
+    #[cfg(feature = "sqlite-mem")]
+    pub use async_sqlite;
+    pub use cosmian_findex;
```

Comment:
I'm not sure re-exporting Findex is a good idea. This will surely lead to importing it twice since client repo will surely import it directly in addition to the memory repo.

Reply:
On client repo, the idea is to import only `findex_memories` in order to use only 1 version of `cosmian_findex` and avoid a potential mix up or a version conflict

Reply:
Then the different README files have to be changed accordingly. It feels strange to import Findex via a memory though. But I can't see strong arguments against it.

Reply:
After letting this thought rest for some time, it seems to me now that the memory ADT should be moved in this repository and the Findex repository should be the one to re-expose the memories. This makes much more sense.